packages/lit-dev-tools-cjs/src/api-docs/types.ts

```diff
@@ -28,14 +28,14 @@ export interface MigrationComment
 
 export interface ExtendedDeclarationReflection extends DeclarationReflection {
   location?: Location;
-  externalLocation?: ExternalLocation;
```

Comment:
Do we need to add `externalUrl` here instead? Maybe not since we're just doing an inline assertion to add anyway, but also maybe yes for documentation purposes.

Reply:
`externalUrl` is already documented natively on `DeclarationReflection`. E.g., it's a TypeDoc type that I am "correcting". However I can add it here for additional documentation.

Reply:
Oh if it's already there the no problem! I was just going based off the diff.